docs/process/prd.md

```diff
@@ -0,0 +1,226 @@
+
+Finally, implementing and maintaining a core component with great performance characteristics, test coverage, and documentation allows us to establish high-quality culture within the project and expect that other components match it.
+
+### Plural Rules
+
+Pluralization is similar to Locale in that it is a core operation required by many other components, it’s a lean module and an ECMA-402 component.
+
+As such it carries similar value and justification for inclusion in 1.0 scope.
+
+### Unicode Set
+
+Unicode Properties are required for many low level components including a number of our business and strategic alignments.
+
+By providing a high quality API backed by a powerful data management solution, ICU4X becomes attractive for segmentation, collation, and unicode regular expression targets which we already see interest from.
+
+Unicode Properties API is also critical for our ability to harness the dynamic Rust community as the GUI/layout needs arise.
+
+### Segmentation
+
+Segmentation aligns business needs, allowing us to attract Mozilla investment and in return getting ability to test our model in production supplying the needs of both Gecko layout engine and ECMA-402 component.
+
+The segmentation API carries also similar strategic value for the project as the Unicode Set as an attractor for the Rust community, but with a higher-level API allows us to validate performance claims in a performance-critical part of all layout needs which also serves our goal of attracting corporate interest in the project.
+
+### DateTimeFormat
+
+Date and Time formatting are the most common high-level requests out of internationalization APIs and a core part of ECMA-402.
+
+On the technical side, this API allows us to validate our value proposition with a high-level, complex, highly demanded API. We have a chance to explore our modularization capabilities, high-volume data management, and non-trivial algorithmic requirements.
+
+On the business side, we get a chance to supply the most common user needs, and cater to the Internationalization and Rust communities.
+
+### NumberFormat
+
+Number formatting represents a blend of low-level foundational APIs such as Locale, Plural Rules and Unicode Set, with high-level APIs such as DateTimeFormat.
+
+It is required for DateTimeFormat, and will exercise our ability to maintain modularity and lean profile of our APIs, while building a large feature set that [ECMA-402 NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) carries.
+
+### FFI/WASM
+
+FFI poses a lot of technical challenges that we have to overcome to validate the business value of ICU4X being a solution to the high-maintenance cost of keeping separate implementations for many target environments.
+
+We’ll need to validate the ability to produce bindings to other programming languages and technically prove that the DataProvider model can fit those requirements.
+
```

Comment:
question: should something related to unicode properties / regexp be on this list?

Reply:
I don't think so, but if you have a change in mind, I'm happy to apply it!